test/Unit.t.sol

```diff
@@ -208,7 +208,7 @@ contract Test_BuybackHook_Unit is TestBaseWorkflow, JBTest {
         // Package data for ruleset call.
         JBRulesetMetadata memory _rulesMetadata = JBRulesetMetadata({
             reservedPercent: JBConstants.MAX_RESERVED_PERCENT,
-            redemptionRate: JBConstants.MAX_REDEMPTION_RATE,
+            cashOutTaxRate: JBConstants.MAX_CASH_OUT_TAX_RATE,
```

Comment:
This should be `0` right? Lots of occurrences where this is `MAX_CASH_OUT_TAX_RATE`

Reply:
oo. nice.